lib/mutant/isolation.rb

```diff
@@ -1,31 +1,35 @@
 module Mutant
   # Module providing isolation
-  module Isolation
+  class Isolation
     Error = Class.new(RuntimeError)
 
-    module None
-
+    class None < self
       # Call block in isolation
       #
       # @return [Object]
       #
       # @raise [Error]
       #   if block terminates abnormal
-      def self.call(&block)
+      def call(&block)
         block.call
```

Comment:
Instead of assigning `&block` explicitly, what about just using `yield` within this method? It's not like you use `block` elsewhere here.